### DISCUSSION TITLE
Is there a way to use markdown-unlit with this plugin?

### DISCUSSION THREAD

**Discussion by kamoshi** [mrcjkb/haskell-tools.nvim#328] [General]
I would like to use [markdown-unlit](https://github.com/sol/markdown-unlit) as the default literate Haskell preprocessor for all standalone `.lhs` files. Is there a way to configure this Neovim plugin, so that it uses markdown-unlit by default?

For example instead of the regular bird-style I would like to write files like this:

```markdown
Some text here

```haskell
fact :: Integer -> Integer
fact 0 = 1
fact n = n * fact (n-1)
\```

Some text after
```

For this to work, I would probably have to pass `-pgmL markdown-unlit` to GHC, but I'm not sure how to do it via plugin settings.

![image](https://github.com/mrcjkb/haskell-tools.nvim/assets/18511281/25c4b8c8-d6da-4c81-b06f-e01c9ce93085)

**Comment by mrcjkb** [comment]
Hey 👋 

This looks like something that you will be more likely to find an answer for in the [haskell-language-server discussions](https://github.com/haskell/haskell-language-server/discussions).

It may be possible to pass GHC flags via the haskell-language-server command (The `hls.cmd` option in this plugin).
See also: `:h haskell-tools.config`.
